pdr_backend/cli/arg_feed.py

```diff
@@ -59,6 +59,18 @@ def __str__(self):
 
         return feed_str
 
+    def output_for_legend(self):
```

Comment:
the name of this function is a bit out of context here. How about str_without_exchange? Otherwise, what legend are we talking about? We have to guess there is some module where there is some legend, while looking just at this file.